bases/br_ms_sinasc/microdados/table_config.yaml

```diff
@@ -121,400 +113,999 @@ partitions: # REQUIRED
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: inicio_pre_natal
+      bigquery_type: string
+      description: Mês de gestação em que iniciou o pré‐natal
+      temporal_coverage:
+          - 2010(1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: month
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: pre_natal
+      bigquery_type: int64
+      description: Número de consultas pré‐natal por semana de gestação
+      temporal_coverage:
+          - 2010(1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: pre_natal_agr
+      bigquery_type: string
+      description: Número de consultas de pré‐natal agrupada
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: classificacao_pre_natal
+      bigquery_type: string
+      description: Códigos de classificação de adequação ao pré natal, gerado pelo
+          sistema
+      temporal_coverage:
+          - 2014(1)
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: quantidade_filhos_vivos
+      bigquery_type: int64
+      description: Número de filhos vivos
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: person
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: quantidade_filhos_mortos
+      bigquery_type: int64
+      description: Número de perdas fetais e abortos
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit: person
+      has_sensitive_data: no
+      observations: A variável aborto é somada com a quantidade de filhos mortos para
+          1994 e 1995
+      is_in_staging: true
+      is_partition: false
+    - name: id_pais_mae
+      bigquery_type: string
+      description: Se a mãe for estrangeira, constará o código do país de nascimento
+      temporal_coverage:
+          - 2010(1)
+      covered_by_dictionary: yes
+      directory_column:
```

Comment:
o código é o ISO-3? Porque se for, tem no nosso diretório mundial

Reply:
Pelo que eu vi é diferente! Esses ids se referem aos estados e o id_pais_residencia indica 1 pro Brasil